src/map/config.js

```diff
@@ -23,7 +23,7 @@ export const MAX_SPRITES_PER_LAYER = 200000
 // ie how large the clicked region should be for including vessels
 export const VESSEL_CLICK_TOLERANCE_PX = 10
 
-export const CLUSTER_CLICK_ZOOM_INCREMENT = 1
+export const CLUSTER_CLICK_ZOOM_INCREMENT = 2
```

Comment:
Does mapbox provide a zoom level where the cluster is going to "expand"? It could be useful instead of this fixed value

Reply:
I saw you could get it with https://docs.mapbox.com/mapbox-gl-js/api/#geojsonsource#getclusterexpansionzoom and it is executed in the code but doesn't seem to be used later.

Reply:
Good catch, thanks. Addressed here: https://github.com/GlobalFishingWatch/map-components/pull/57/commits/30d3b018692f9ef22483bb7d5ee110a444dcca28